swingset-demo/src/main/java/com/nqadmin/swingset/demo/MainClass.java

```diff
@@ -651,6 +654,7 @@ public static void main(final String[] _args) {
 				case 's': userSqlFiles.add(g.getOptarg());  break;
 				case '?':
 				case 'h':
+				default: // TODO: not sure if default is OK here meaning all other characters indicate an error??
 					some_error = true;
```

Comment:
It's cool. g.getopt() never returns anything not in its list. But adding default would catch forgetting to add a case for a new option, rather than silently ignoring.